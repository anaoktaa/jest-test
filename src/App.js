import React from 'react';
import reactlogo from './assets/images/react-logo.png';
import './App.css';

const App = () => {
    return (
        <div className='app'>
            <h1>Welcome to React Application</h1>
            <img src={reactlogo} alt=''/>
        </div>
    );
}

export default App;