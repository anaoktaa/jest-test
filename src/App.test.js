import React from 'react';
import { shallow, mount, render } from 'enzyme';
import App from './App';

it('expect to render app component', () => {
    expect(shallow(<App/>)).toMatchSnapshot();
});


