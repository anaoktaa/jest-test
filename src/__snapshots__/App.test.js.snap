// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render app component 1`] = `
<div
  className="app"
>
  <h1>
    Welcome to React Application diperbaharui lagi
  </h1>
  <img
    alt=""
    src="test-file-stub"
  />
</div>
`;
